guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,607 @@
 {
```

Comment:
<p>Can we call out the updates to API ref here. We should be able to merge for Widgets and Features modules</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yep - added a note about adding text emphasis, links to returned objects, and code snippets to `arcgis.geometry`, `arcgis.features`, and `arcgis.widgets`